lib/content._coffee

```diff
@@ -47,11 +44,14 @@ getFilePathForId = (id) ->
     id = id.replace /([A-Z])/g, '_$1'
     Path.join DIR_BY_ID_PATH, "#{id}.json"
 
-getFilePathForURL = (url) ->
-    if idFile = DATABASE["#{hashURL url}.json"]
-        Path.join DIR_BY_ID_PATH, idFile.replace '../content-by-id', ''
-    else
-        ''
+readFileForURL = (url, _) ->
+    cloudClient.download
+        container: 'content/content-by-url'
+        remote: "#{hashURL url}.txt"
```

Comment:
When I tested the raw call, including `content-by-url` in the `container` worked. I wouldn't be surprised if it also works to put it in the `remote`, and if it's more correct to do that I'll go ahead and change it. 

I don't think I'll add a config yet, until we figure out how we want to separate it for development.